plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -233,6 +234,16 @@ class NewCreateServiceModalForm extends Component {
     this.setState({batch, appConfig: this.getAppConfig(batch)});
   }
 
+  handleSetItem({value, path}) {
```

Comment:
Ideally, I would love if I could bubble a form event originating from a button. This can be proven particularly useful in various cases.

Keeping a local state means that we need some extra effort for properly maintaining and synchronising it with the form state, that I would prefer to avoid.

What do you think?